Meadowlark-js/backends/meadowlark-postgresql-backend/test/integration/Update.test.ts

```diff
@@ -755,13 +754,10 @@ describe('given the update of an existing document changing meadowlarkId with al
 
     expect(result.rowCount).toEqual(0);
   });
-  it('should have deleted the document reference  related to the old meadowlarkId in the db', async () => {
-    const findParentReferenceByMeadowlarkIdSql = format(
-      `SELECT alias_id FROM meadowlark.aliases WHERE document_id = %L FOR SHARE NOWAIT`,
-      meadowlarkId,
-    );
-    const result: any = await client.query(findParentReferenceByMeadowlarkIdSql);
 
-    expect(result.rowCount).toEqual(0);
```

Comment:
Why remove this old test?

Reply:
Because it is duplicated. The previous one is executing the same test.